api-spec/README.md

```diff
@@ -1,100 +1,55 @@
+A STAC API is the dynamic version of a SpatioTemporal Asset Catalog. It returns STAC [Catalogs](../catalog-spec/README.md), [Collections](../collection-spec/README.md), or [Items](../item-spec/README.md), depending on the endpoint. Catalogs and Collections are JSON, while Items are GeoJSON - Features when just a single Item, or a Feature Collection when multiple Items are returned from a search.
 
-The core of the spec is a single endpoint:
+The API is a [Web Feature Service 3.0 (WFS 3) API](https://github.com/opengeospatial/WFS_FES), in that WFS 3 defines many of the endpoints that STAC uses. A STAC API should be compatible and usable with WFS3 clients. However, WFS 3 is still under development and while STAC tries to stay in sync with WFS3 developments, there may be discrepencies prior to final versions of both specifications.
 
-```
-/stac/search
-```
-
-It takes a JSON object that can filter on date and time:
-
-```json
-{
-  "bbox": [5.5, 46, 8, 47.4],
-  "time": "2018-02-12T00:00:00Z/2018-03-18T12:31:12Z"
-}
-```
-
-This tells the server to return all the catalog items it has that are from the second half of March, 2018 and
-that intersect with this area:
-
-![swiss_bbox](https://user-images.githubusercontent.com/407017/38382405-b5e69344-38be-11e8-90dc-35738678356d.png)
-
-_Map © OpenStreetMap contributors_
-
-The return format is a [GeoJSON](http://geojson.org/) `FeatureCollection` with features compliant with the
-[Item spec](../item-spec/README.md) for STAC. It returns to a limit optionally requested by the client, and includes
-pageable links to iterate through any results past that limit.
+## In this directory
 
-## Dynamic Catalog API
+**The Specification:** The main definition of the STAC API specification is in the *[api-spec.md](api-spec.md)* file. It includes an overview and in depth explanation of the REST endpoints and parameters.
 
-The other endpoint that is included as an option in STAC API is `/stac/`, which implements the [STAC Catalog Spec](../catalog-spec/README.md).
-Implementing this enables tools like
-[STAC Browser](https://medium.com/@mojodna/a-stac-browser-348a60674061) to use the dynamic catalog, to enable better
-discovery through people browsing and search engines crawling.
+**Extensions:** API Extensions are given in the *[extensions](extensions/)* folder. YAML fragments are provided for each extension with details provided in the *[README](extensions/README.md)*.
 
-The OpenAPI spec in this directory documents the endpoint, and refer to the STAC Catalog and [STAC Collection](../collection-spec/README.md) for more information about the full content and link structure.
+**Examples:** For samples of how the STAC API can be queried, see the *[examples.md](examples.md)* file.
 
-## API Fragments
+**Schemas:** The API is described by the OpenAPI specification documents in the *[openapi](openapi/)* folder.

```

Comment:
I don't think we need to call this 'schemas', as they aren't really schemas in the sense of JSON or XML schemas. I think we can just say 'OpenAPI documents' or 'OpenAPI fragments', etc.

Reply:
From https://swagger.io/docs/specification/about/ it seems like maybe the best term is 'OpenAPI description'.